sale_order_line_packaging_qty/tests/test_sale_order_line_packaging_qty.py

```diff
@@ -30,3 +30,58 @@ def test_product_packaging_qty(self):
         self.assertEqual(order_line.product_packaging_qty, 1.0)
         order_line.write({"product_packaging_qty": 3.0})
         self.assertEqual(order_line.product_uom_qty, 15.0)
+
+    def test_onchange_qty_is_pack_multiple(self):
+        order = self.env["sale.order"].create({"partner_id": self.partner.id})
+        order_line = self.env["sale.order.line"].create(
+            {
+                "order_id": order.id,
+                "product_id": self.product.id,
+                "product_uom": self.product.uom_id.id,
+            }
+        )
+        self.assertFalse(order_line._onchange_product_uom_qty())
+
+        self.product.write({"only_sell_by_pack": True})
+        self.assertTrue(order_line._onchange_product_uom_qty())
+
+        order_line.product_id_change()
+        self.assertFalse(order_line._onchange_product_uom_qty())
+
+        order_line.write({"product_uom_qty": 3.0})
+        self.assertTrue(order_line._onchange_product_uom_qty())
+
+        order_line.write({"product_uom_qty": self.packaging.qty * 2})
+        self.assertFalse(order_line._onchange_product_uom_qty())
+
+    # def test_auto_fill_packaging(self):
```

Comment:
Remove what you don't use

Reply:
I will fix it's it's reason why I added on WIP on the description :wink: